main.py

```diff
@@ -38,18 +35,35 @@ def inside_trades():
 
     buy, sell = transactions.get_trade_decisions()
     orders = helpers.place_trades(buy, sell)
-    helpers.activate_order(orders)
+    helpers.activate_orders(orders)
+
+
+def schedule_trades_for_year():
+    opening_days = helpers.get_opening_days()
+
+    for i in range(len(opening_days)):
+        scheduler.add_job(inside_trades,
+                          trigger=CronTrigger(month=opening_days[i].month,
+                                              day=opening_days[i].day,
+                                              hour=13,
+                                              minute=9,
+                                              timezone=utc))
 
 
 if __name__ == '__main__':
     scheduler = BlockingScheduler(timezone=utc)
 
-    scheduler.add_job(inside_trades,
-                      trigger=CronTrigger(day_of_week="mon-fri",
-                                          hour=10,
-                                          minute=30,
-                                          timezone=utc),
-                      name="Perform inside trades")
+    inside_trades()
```

Comment:
Good point, this was just for testing purposes (otherwise would have had to change the time on the scheduler constantly for stuff to run). Left it in because I thought we might need that for demo purposes if we ever update the YouTube videos.